### ISSUE TITLE
Duplicate navigation

### ISSUE THREAD

**Issue by Arnopello** [BeSite/jQuery.TosRUs#73]
Hi

I pointed out a new problem, probably linked to my under-knowledge in JS. 
See on the following page : http://dp.lmqsrv.lumeeq.com/photos/
I declare TosRus "for each container with a specific class". Currently, there are two of them in that page. The problem is we can see in the code (ans briefly while loading the page on a tablet) that there are two navigations. 
I think it's due to the settings being created "for each container", thus creating as much navigation bar as there are containers. 

Can you help me or give me an insight of how to solve it?

here is my declaration:

jQuery(function($){
   $(document).ready(function() {
      $(".foogallery-container").each(function() {
  $(this).find( "a" ).tosrus({
pagination : {
 add : true,
 type : "thumbnails"
   },
wrapper : {
onClick : "close"
},
buttons : {
prev : true,
next : true,
close : true
}
});
   });
});
})

**Comment by FrDH** [comment]
I think the issue is you're including both the ".all" and the ".pagination" js files.
This adds the pagination functionality twice.

**Comment by Arnopello** [comment]
Hi

Thanks for your fast answer. I took the .pagination out and it worked! I also took out the .pagination CSS out and it seems to be ok. 

Does that mean that .all include all plugins?

For example. In the "options" section of TosRus website, you say for Drag "Note: These options require the drag add-on and the Hammer.js touch-framework to be included.". But, if I included jquery.tosrus.min.all.js, drag add-on as well as pagination add-on are already included?

Thanks
